### PR TITLE
Root function

### PR DESCRIPTION
**Pull request by sarcar**
Added example of Quadratic Equation to the section on Functions. I have kept the material on Temperature conversion as well for completeness; during class the instructor can choose to jump over to the Quadratic section is s/he so desires.

### CODE REVIEW

File: Day3/TemperatureConversion.md [hunk 1]
```diff
@@ -44,6 +44,112 @@ Create a new file, temperatureConversion.py. Copy the function definition code a
...
+roots = quad_roots(a, b, c)
+print("Roots are:", roots[0], roots[1])
+```
+
+
+
+Test your program with the following inputs : a = 1, b = -8, c = 15. You should get roots of 5.0 and 3.0 (similar to what is shown below)
+
+```
+C:\Users\305002615\AppData\Local\Continuum\Anaconda3\python.exe C:/SHOURYA/DEV/PYTHON/summerschool/quadroots.py
```
Comment (graichen) [inline Day3/TemperatureConversion.md]: Since we are running in pycharm, let's remove the command line from your output.

**Comment by graichen** [comment]
Since the exercises at the bottom are related to the temperature, should they move up
Should we add exercises related to the main program section that build off of both functions?

File: Day3/TemperatureConversion.md [hunk 2]
```diff
@@ -44,6 +44,112 @@ Create a new file, temperatureConversion.py. Copy the function definition code a
...
+b = float(b)
+c = float(c)
+
+roots = quad_roots(a, b, c)
+print("Roots are:", roots[0], roots[1])
+```
+
+
+
+Test your program with the following inputs : a = 1, b = -8, c = 15. You should get roots of 5.0 and 3.0 (similar to what is shown below)
```
Comment (graichen) [inline Day3/TemperatureConversion.md]: Label this section as an Exercise, so it is easy for students to find the "work".

File: Day3/TemperatureConversion.md (hunk 1 above)
Comment (sarcar) [reply Day3/TemperatureConversion.md]: The commandline is a part of the PyCharm console output , but I can remove it.

File: Day3/TemperatureConversion.md (hunk 2 above)
Comment (sarcar) [reply Day3/TemperatureConversion.md]: Will do.

File: Day3/TemperatureConversion.md [hunk 3]
```diff
@@ -170,8 +170,25 @@ if __name__ == "__main__":
    Divide the function and main into two python files and run again.
 
 4. Run this program and see the different output. 
+
 5. Write a program that calls two functions ```simpint()``` and ```compint()``` to calculate the Simple Interest and Compound Interest on a (common) Principal Amount, Rate of Interest and Number of Years. Print both the interest values and the difference between them.
 
+   $Simple Interest = \frac{P * R * N}{100}$
```
Comment (graichen) [inline Day3/TemperatureConversion.md]: this does not seem to be the correct formatting for markdown.

**Pull request merged**